app/views/layouts/listnav/_ems_storage.html.haml

```diff
@@ -6,13 +6,13 @@
     = miq_accordion_panel(_("Properties"), false, "ems_prop") do
       %ul.nav.nav-pills.nav-stacked
         = li_link(:if => true,
-          :text       =>_('Summary'),
+          :text       => _('Summary'),
           :record_id  => @record.id,
           :display    => 'main',
           :title      => _("Show Summary"))
-        = li_link(:if => (@record.has_events? || @record.has_events?(:policy_events)),
```

Comment:
@hstastna yes, both link & button should check for events availability. Need to add a check similar to a check for Timeline button on Cluster summary screen https://github.com/ManageIQ/manageiq/blob/master/app/helpers/application_helper/toolbar_builder.rb#L707